### PR TITLE
[Tizen] Build chip-tool using build_examples.py

### PR DESCRIPTION
**Pull request by arkq**
#### Problem

Easy way to build chip-tool for Tizen platform.

#### Change overview

- added chip-tool for Tizen to build_examples.py script

#### Testing

```
./scripts/build/build_examples.py --target tizen-arm-chip-tool build
```

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
Fast tracking platform changes

File: .github/workflows/examples-tizen.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-tizen:0.5.79
+            image: connectedhomeip/chip-build-tizen:0.5.81
```
Comment (andy31415) [inline .github/workflows/examples-tizen.yaml:35]: I believe updates should generally be done globally (change all workflows).
This makes it easier to ensure that images are consistent (and that chip-build-vscode we use can compile everything)

Comment (Arkq) [reply .github/workflows/examples-tizen.yaml:35]: OK, so this PR can wait for #19937 to get merged

Comment (andy31415) [reply .github/workflows/examples-tizen.yaml:35]: We seem to have trouble with Infineon image builds. Resolving this comment -we can move everything in sync once build issues are resolved.

**Pull request merged**